pyproject.toml

```diff
@@ -0,0 +1,160 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "morphosource"
+dynamic = ["version"]
+description = 'Python package for interacting with morphosource.org API'
+readme = "README.md"
+requires-python = ">=3.7"
+license = "MIT"
+keywords = []
+authors = [
+  { name = "John Bradley", email = "johnbradley2008@gmail.com" },
+]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+]
+dependencies = [
+  "requests",
+  "pygbif"
+]
+
+[project.urls]
+Documentation = "https://github.com/unknown/morphosource#readme"
```

Comment:
Why is it an unknown repo base for a repo called morphosource instead of linking to Imageomics/pyMorphoSource?

Reply:
I used hatch to generate the config. Perhaps there is some flag I missed in the generate process. I'll fix the URL.